Add a "What's new" section to the Mod Docs standard (#214)
redhat-documentation/modular-docs: modular-docs-manual/content/topics/whats-new.adoc

[id="whats-new_{context}"]
= What's new

> The following list summarizes significant changes to the mod docs standard over time.

Reviewer: _mod docs_ seems like an internal reference, consider using _modular documentation_ instead.
```suggestion
The following list summarizes significant changes to the modular documentation standard over time.
```

> Apply these changes when you create and revise content.

Reviewer 2: ```suggestion
Apply these changes only when you create and revise content.
```

Added only so the next sentence makes sense, but am open to other suggestions.

Author: Thanks for the suggestion, @ncbaratta. Updated.

[id="whats-new_{context}"]
= What's new

The following list summarizes significant changes to the mod docs standard over time.

> In most cases, apply these changes to your new content and revisions.

Reviewer 2: I'm not sure what you're trying to say - with the next line I think you're saying you don't have to change anything in existing content, just new. Which means we don't need this line at all.

Author: Thanks. I hope the updated version is clearer.

It is not expected that you apply changes retrospectively to existing content.

> In rare cases where all content changes need to be made, the Mod Docs team will communicate this information through CCS leadership.

Reviewer 2: ```suggestion
CCS leadership will communicate in instances where changes to all content will need to be made.
```

Author: Thanks. Updated.

> // Template:

Reviewer 2: ```suggestion
// Release Notes Template:
```

Author: Thanks. Updated.
